src/config.rs

```diff
@@ -43,6 +43,8 @@ pub struct Config {
     // defaults to `corner_r`
     pub padding: Option<f64>,
 
+    pub inhibit_compositor_keyboard_shortcuts: bool,
```

Comment:
Should be mentioned in readme's example config.

Reply:
Done.